src/Command/CompileCommand.php

```diff
@@ -62,7 +62,7 @@ protected function executeAll(Arguments $args, ConsoleIo $io): int
     {
         $io->info('Compiling all templates');
 
-        foreach (Scanner::all() as $section => $templates) {
+        foreach (Scanner::all($this->twigView->getExtensions()) as $section => $templates) {
```

Comment:
I just realized, this TwigView instance isn't customizable. How will we run the command with the user's extensions/loader?

Reply:
We could add an option to specify the view class to use.

Reply:
Added `--view` option to specify View class.